index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
En quelle langue faisons-nous le site? Français, non?
Si oui, on peut vite-fait penser à changer la langue ici :
```suggestion
<html lang="fr">
```